src/main/proto/admin.proto

```diff
@@ -256,4 +257,14 @@ message EventProcessorSegment {
   int64 token_position = 6;
   bool is_in_error = 7;
   string error = 8;
+}
+
+message LoadBalanceRequest {
+  EventProcessorIdentifier processor = 1;
+  LoadBalanceStrategy strategy = 2;
+}
+
+enum LoadBalanceStrategy {
```

Comment:
Don't use an enum for this. If we introduce a new strategy this would require an API change.